src/main/java/hw3/components/Frame.java

```diff
@@ -0,0 +1,38 @@
+package hw3.components;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class Frame extends AbstractPageComposite {
```

Comment:
It will be better specify fore which frame this class. We have 3 frames on the page

Reply:
Sure, renamed.